src/about-us/index.html

```diff
@@ -0,0 +1,189 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block title -%}
+    About Us
+{%- endblock %}
+
+
+{% block content_main %}
+    <section class="block
+                    block__flush-top">
+        <h1>About Us</h1>
+        <p class="h3">
+            We're a new government agency created
+            after the financial crisis of 2008.
+            We write and enforce rules to ensure that markets for
+            consumer financial products and services are fair and open,
+            and to make financial information easier to understand.
+            We also build tools and resources that help arm consumers
+            with the information they need,
+            and empower consumers to make the best financial decisions.
+        </p>
+        <ul class="list list__links">
+            <li class="list_item">
+                <a href="/the-bureau/" class="list_link">
+                    Learn more about the Bureau
+                </a>
+            </li>
+            <li class="list_item">
+                {# TODO: Add link to Consumer Guides #}
+                <a class="list_link u-link__disabled">
+                    Find resources and tools for consumers
+                </a>
+            </li>
+            <li class="list_item">
+                <a href="/offices/payments-to-harmed-consumers/"
+                   class="list_link">
+                    Find out about our work to protect consumers
+                </a>
+            </li>
+        </ul>
+    </section>
+    <div class="content-l">
+        {# The extra div is to prevent border tapering from combining block__border and content-l classes #}
+        <div class="content-l_col-1-2">
+            <section class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-bottom">
+                <h2>CFPB Leadership</h2>
+                <div class="media bureau-bio u-mb30">
```

Comment:
`media bureau-bio block block__flush-top block__flush-bottom block__padded-bottom` would be analogous. Certainly that feels way over-the-top as a replacement for the `u-mb30` utility class, but ideally we could define things as blocks consistently without the need for utility tweaks in certain areas. Not suggesting we update this now, but it may make sense to just keep in mind where we have certain types of blocks that may make sense to create a custom block type for in CF, such as something like `media bureau-bio block block__mb`. Maybe @Scotchester has ideas on this?